client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -83,7 +83,7 @@ export default function getThankYouPageUrl( {
 	isJetpackCheckout?: boolean;
 	jetpackTemporarySiteId?: string;
 	adminPageRedirect?: string;
-	domains: SiteDomain[] | undefined;
+	domains?: SiteDomain[];
```

Comment:
when _everything_ is optional it can be helpful to make a type for the props and then wrap in `Partial<>` - can be much clearer